app/views/assets/index.html.erb

```diff
@@ -12,44 +12,51 @@
 
 <%= content_for :right do %>
   <% if authorized? && @user.assets_count > 2 %>
-
-  <div id="share_user_latest" class="box">
-  <h2 class="box static">Embed Your Latest Music</h2>
-  <div class="static_content">
-    <h3>Embed a flash player by copying and pasting:</h3>
-
-    <%= text_area_tag "flash_player_#{@user.id.to_s}", "<embed src=\"#{FLASH_PLAYER}\" width=\"400\" height=\"140\" allowfullscreen=\"true\" allowscriptaccess=\"always\" flashvars=\"&file=#{user_url(@user.login, format: :xml)}&height=140&width=400&frontcolor=0x3C3C3C&backcolor=0xf3f3f3&lightcolor=0xFF944B&screencolor=0xFF944B&displaywidth=120&showdigits=false&showdownload=true\" />", rows: 3, cols: 50 %>
-    (customize how it looks by visiting <%= link_to 'the setup wizard', 'http://www.jeroenwijering.com/?page=wizard&example=2'%>)
-  </div>
-  </div>
-
-  <div class="box">
-  <h2 class="box">Hot Tracks this week</h2>
-  <div class="static_content">
-      <%# image_tag @user.most_popular_chart %>
-  </div>
-  </div>
-
-  <div class="box">
-  <h2 class="box">Most Favorited Tracks</h2>
-  <div class="static_content">
-      <%# image_tag @user.most_favorited_chart %>
-  </div>
-  </div>
-
-
-  <div class="box">
-  <h2 class="box">Most Commented On Tracks</h2>
-  <div class="static_content">
-      <%# image_tag @user.most_commented_on_chart %>
-  </div>
-  </div>
-
-  <div class="box">
-  <h2 class="box">Most Listened To Tracks (ever)</h2>
-  <div class="static_content">
-      <%# image_tag @user.most_listened_to_ever_chart %>
-  </div>
-  </div>
+    <div id="share_user_latest" class="box">
+      <h2 class="box static">Embed Your Latest Music</h2>
+      <div class="static_content">
+        <h3>Embed a flash player by copying and pasting:</h3>
+
+        <%= text_area_tag "flash_player_#{@user.id.to_s}", "<embed src=\"#{FLASH_PLAYER}\" width=\"400\" height=\"140\" allowfullscreen=\"true\" allowscriptaccess=\"always\" flashvars=\"&file=#{user_url(@user.login, format: :xml)}&height=140&width=400&frontcolor=0x3C3C3C&backcolor=0xf3f3f3&lightcolor=0xFF944B&screencolor=0xFF944B&displaywidth=120&showdigits=false&showdownload=true\" />", rows: 3, cols: 50 %>
+        (customize how it looks by visiting <%= link_to 'the setup wizard', 'http://www.jeroenwijering.com/?page=wizard&example=2'%>)
+      </div>
```

Comment:
We can actually get rid of this whole "Embed Your Latest Music" — it was for an old embeddable player from years ago!